javascript-prep/src/css/detail.css

```diff
@@ -1,19 +1,25 @@
 .detail-container{
```

Comment:
Let's also add a min-height of '100vh'. Otherwise the background might finish before full-screen in some rare cases

Reply:
done